device/src/async_device/mod.rs

```diff
@@ -34,7 +35,7 @@ use core::cmp::min;
 /// - C: A CryptoFactory implementation
 /// - RNG: A random number generator implementation. An external RNG may be provided, or you may use a builtin PRNG by
 /// providing a random seed.
-pub struct Device<R, C, T, G, const N: usize = 256>
+pub struct Device<R, C, T, G, const N: usize = 256, const D: usize = 4>
```

Comment:
Probably makes sense to default to 1 to avoid accidentally increasing memory for existing users

Reply:
Good point! Thanks